frontend/public/components/utils/list-dropdown.jsx

```diff
@@ -175,22 +182,79 @@ ListDropdown.propTypes = {
   id: PropTypes.string,
 };
 
-const NsDropdown_ = (props) => {
-  const openshiftFlag = props.flags[FLAGS.OPENSHIFT];
-  if (flagPending(openshiftFlag)) {
-    return null;
+const useProjectOrNamespaceModel = () => {
+  const nsModel = modelFor('Namespace');
+  const canCreateNamespace = useAccessReview({
+    group: nsModel.apiGroup,
+    resource: nsModel.plural,
```

Comment:
As stated, just import the static NamespaceModel, then use it here:

```suggestion
    group: NamespaceModel.apiGroup,
    resource: NamespaceModel.plural,
```